collectors/python.d.plugin/freeswitch/README.md

```diff
@@ -0,0 +1,61 @@
+# freeswitch
```

Comment:
If I'm not mistaken, the project brands itself as **FreeSWITCH**. We should follow that convention.

```suggestion
# FreeSWITCH
```